### ISSUE TITLE
[2.0] added node does not work properly.

### ISSUE THREAD

**Issue by xal3xhx** [PufferPanel/PufferPanel#913]
tl;dr, i created a new node, the server installs and starts on it but will always show up as offline and console does not work.

i have 2 server pc's i have running locally. one hosts the main pufferpanel webserver and the other is configured as a node. the node config worked perfectly well and i am able to create a server on it (in this case minecraft). the server will install and run perfectly fine from the main panel control but as soon as i refresh the page the server displays offline and i can not enter any commands to the server. the time on the cpu and memory graph continues to move but there is no other data.  the file manager appears to work perfectly fine as well.
